lua/plenary/functional.lua

```diff
@@ -2,7 +2,7 @@ package.loaded['plenary.functional'] = nil
 
 local f = {}
 
-f.map = function(fun, iter)
+function f.map(fun, iter)
```

Comment:
Figured I'd go ahead and make the style consistent. I actually do this in #9, but I'll just go ahead and do it here since it probably makes more sense to do it in this PR anyways.

Reply:
Sounds good.